lib/order.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/menu'
+class Order
+  attr_reader :order_price, :valid_item
+  def initialize
```

Comment:
Could use dependency injection here as to not hardcode the order class to menu and basket.